src/server/scripts/Spells/spell_warlock.cpp

```diff
@@ -306,6 +309,30 @@ class spell_warl_burning_rush_aura : public AuraScript
     }
 };
 
+// 104316 - Call Dreadstalkers
+class spell_warl_call_dreadstalkers : public SpellScript
+{
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_WARLOCK_CALL_DREADSTALKERS_LEFT, SPELL_WARLOCK_CALL_DREADSTALKERS_RIGHT });
+    }
+
+    void HandleAfterCast() const
+    {
+        Unit* caster = GetCaster();
+        CastSpellExtraArgs args;
+        args.SetTriggerFlags(TRIGGERED_IGNORE_CAST_IN_PROGRESS | TRIGGERED_DONT_REPORT_CAST_ERROR);
+        args.SetTriggeringSpell(GetSpell());
+        caster->CastSpell(caster, SPELL_WARLOCK_CALL_DREADSTALKERS_LEFT, args);
+        caster->CastSpell(caster, SPELL_WARLOCK_CALL_DREADSTALKERS_RIGHT, args);
+    }
+
+    void Register() override
+    {
+        AfterCast += SpellCastFn(spell_warl_call_dreadstalkers::HandleAfterCast);
```

Comment:
I would also use OnEffectHitTarget instead. The whole point of the effect to be using TARGET_UNIT_TARGET_ENEMY is to pass information to these summons as to whom they should charge.